qa/smoke-test-tribe-node/build.gradle

```diff
@@ -17,53 +17,27 @@
  * under the License.
  */
 
-import org.elasticsearch.gradle.test.ClusterConfiguration
-import org.elasticsearch.gradle.test.ClusterFormationTasks
-import org.elasticsearch.gradle.test.NodeInfo
+apply plugin: 'elasticsearch.multi-cluster-test'
 
-apply plugin: 'elasticsearch.rest-test'
-
-List<NodeInfo> oneNodes
-
-task setupClusterOne(type: DefaultTask) {
-  mustRunAfter(precommit)
-  ClusterConfiguration configOne = new ClusterConfiguration(project)
-  configOne.clusterName = 'one'
-  configOne.setting('node.name', 'one')
-  oneNodes = ClusterFormationTasks.setup(project, setupClusterOne, configOne)
-}
-
-List<NodeInfo> twoNodes
-
-task setupClusterTwo(type: DefaultTask) {
-  mustRunAfter(precommit)
-  ClusterConfiguration configTwo = new ClusterConfiguration(project)
-  configTwo.clusterName = 'two'
-  configTwo.setting('node.name', 'two')
-  twoNodes = ClusterFormationTasks.setup(project, setupClusterTwo, configTwo)
+clusters {
```

Comment:
I wonder if this should be `extraClusters` to make it clear that these are in addition to (not replacing) the cluster that you get from `integTest`.